src/types.ts

```diff
@@ -177,7 +184,7 @@ export interface Options {
   wrapperClasses?: string | string[] | undefined | null | ((id: string, code: string) => string | string[] | undefined | null)
 
   /**
-   * Component name to wrapper with
+   * Component name to wrap with (will wrap the wrapperDiv if present)
```

Comment:
What does this mean?

Reply:
I wanted to clarify the relationship between wrapperDiv and wrapperComponent. i.e. explain that if wrapperDiv is set to true and a wrapperComponent is defined, the output will look like this:

```html
<WrapperComponent>
  <div class='markdown-body'>
    [markdown here]
  </div>
</WrapperComponent>
```

We could maybe expand the text in the brackets to 'this will wrap the wrapper div when wrapperDiv=true'

Alternatively, we could remove if it is not worth mentioning (and assume that it is fairly easy to work out what options will generate the desired behaviour.

Reply:
I see, if you mean on the wrapping order, I think it's fine to keep as-is, as it doesn't make much sense to wrap `<div` outside of the components anyway.

Reply:
Thanks for merging :)